packages/apps/job-launcher/server/src/modules/user/user.service.spec.ts

```diff
@@ -13,6 +13,7 @@ import { UserStatus, UserType } from '../../common/enums/user';
 import { ethers } from 'ethers';
 import { IUserBalance } from '../../common/interfaces';
 import { Currency } from '../../common/enums/payment';
+import * as bcrypt from 'bcrypt';
 
 const PASSWORD_SECRET = '$2b$10$EICgM2wYixoJisgqckU9gu';
```

Comment:
Perhaps we can use mockup variables for the data used in the tests from `tests/constants`. Especially those that are repeated, such as user emails, passwords, etc. Just proposal (low prior).

Reply:
You are right, we will have to do a tests rafactoring in the future, to fix this kind of things and also make them more efficient, at this moment they take too long.